.gitignore

```diff
@@ -25,6 +25,8 @@ DerivedData
 *.ipa
 *.xcuserstate
 project.xcworkspace
+ios/Pods
+Podfile.lock
```

Comment:
How does the `Podfile.lock` work? Is it the same as for example `yarn.lock`? In this case we should probably commit it.

Reply:
I saw in the repo for the native iOS app, their gitignore contained Podfile.lock. That is why I added it

Reply:
Ok, this comment is all I need to know... 😊